### DISCUSSION TITLE
NextAuth cookies are stripped from PUT requests in production (Safari only)

### DISCUSSION THREAD

**Discussion by traintime-de** [nextauthjs/next-auth#2680] [Help]
### Question 💬

We seem to be having a problem with next-auth where cookies are stripped from PUT requests in production, although is working locally.

Our setup:

- Next 10.0.6
- Next-Auth: 3.29.0

The issue seems to occur across Safari / Webkit based browsers

Any advice greatly appreciated - have been working on this issue for weeks!!!

Thanks,
Steve

### How to reproduce ☕️

**Safari locally:**
![image](https://user-images.githubusercontent.com/78261764/132247538-fca98bd4-6ae8-4efb-bd07-1b0404320a47.png)

**Safari production:**
![image](https://user-images.githubusercontent.com/78261764/132247609-5a406e4a-1ffc-4c52-be0a-be22c852f6af.png)

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by traintime-de** [comment]
This is the error is the in the Safari console:

![image](https://user-images.githubusercontent.com/78261764/132247895-b382e92c-7e32-4817-9917-9f02d8f8cd67.png)

**Comment by balazsorban44** [comment]
It doesn't sound like a nextauth specific issue to me. can you confirm that it's only nextauth cookies that are being stripped?

do you have a reproduction?
